README.md

```diff
@@ -178,7 +178,7 @@ fork {
     int emiratesPrice;
 
     // Get the response and price for airline 'Qatar Airways'
-    if (airlineResponses["qatarWorker"] != null) {
+    if (airlineResponses["qatarWorker"] != ()) {
```

Comment:
Shall we use nil lifting or match statements instead of checking using if statements in all places?

Reply:
We can actually eliminate this check in all the `fork-join` operations those join all the results. However it is needed for the conditional join operation to check which worker responses first.

And in the json parsing validations, if we are to match each element separately we will end up duplicating the code segment that sends a response with `bad request` message. Hence, it's checked using if conditions only after parsing all the elements.